.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  allow_failures:
+    - php: 7.0
+    - php: hhvm
 
 before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev --no-interaction
+  - composer self-update
+  - composer install --prefer-source --no-interaction
 
 script:
   - mkdir -p build/logs
   - mkdir -p build/cov
-  - php bin/phpunit -c phpunit.xml.dist
+  - phpunit
```

Comment:
FYI, we target the a >= 4.6.4, stable , up to date version of PHPUnit (See recent #17 & #20).
You're choosing to bypass the PHPUnit vendor's version and preferring the Travis built-in one.

Could you elaborate a bit on that choice ?
Why not changing the composer.json package list then ?